railgun/test/integration/suite_test.go

```diff
@@ -51,6 +52,9 @@ const (
 
 	// ingressClass indicates the ingress class name which the tests will use for supported object reconcilation
 	ingressClass = "kongtests"
+
+	// elsewhere is the name of an alternative namespace
+	elsewhere = "elsewhere"
```

Comment:
It's not clear to me why we have this as a glob in the test suite as opposed to in the multi-namespaces test :thinking: an you please help me to understand?

Reply:
I initially thought that starting the controller with that namespace in the watch list would actually prevent startup, but it looks like that's not the case. Creating it in the test after the controller starts works fine.

The other case to make for putting it in suite is that we do still have to include it in the watch namespace list there, and it feels a bit weird to say "yes, we'll watch it, but we don't know if it exists, up to tests to create it". That's a subjective call though, it doesn't break anything as long as _something_ creates it. WDYT?

Reply:
:thinking: I'm leaning towards creating it just before the tests that need it, and then tearing it down in a defer in that tests just because it's not clear to me that anything else needs this namespace for now. However, if you have a sense that this can be used more in other places later and want to keep it as a glob, I don't consider this a blocker let's just make sure we add more comments that explain how it's being used.